docs/guides/qiskit-function-templates.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Qiskit Function templates
+description: Learn how to use Qiskit Code Assistant, a generative AI code assistant.
+---
+{/* cspell:ignore FCIDUMP */}
+
+
+# Introduction to Qiskit Function templates
+
+The [Qiskit Function templates](https://github.com/qiskit-community/qiskit-function-templates) provide realistic examples for how to build utility-scale application workflows with Qiskit and the Qiskit addons. These prebuilt workflows are customizable for a variety of research needs and can be used instead of starting from scratch. To simplify execution and parallelization, they are easily deployable to [Qiskit Serverless](./serverless).
+
+The templates provided are broken into two basic categories:
+
+- *Application Templates*: Templates to build realistic research workflows.
+- *Base Templates*: Basic implementation use for reference in the development process.
+
+## Application template implementations
+
+The application template implementations are organized by application area, which currently includes a physics and chemistry application utilizing the [SQD addon](./qiskit-addons-sqd) and the integral equation formalism polarizable continuum model (IEF-PCM).
+
+For convenience, direct download links are provided for each application template directory:
+
+- [Download Hamiltonian Simulation template](https://ibm.biz/ham-sim-template)
+- [Download SQD IEF-PCM template](https://ibm.biz/sqd-pcm-template)
+
+### Chemistry Workflow
+
+This template encapsulates a workflow for quantum-centric implicit solvent simulations of ground state energy in molecular systems. More detail about the workflow's methodology can be found in the template's [README.md documentation](https://github.com/qiskit-community/qiskit-function-templates/blob/main/chemistry/sqd_pcm/README.md) and a Jupyter notebook describing how to deploy and run this template can be found in the [same directory](https://github.com/qiskit-community/qiskit-function-templates/blob/main/chemistry/sqd_pcm/deploy_and_run.ipynb). It is composed of three sections described below:
```

Comment:
I think this section have a few details on interesting things users should experiment with, in line with "customisable pipelines", where you can replace components yourself (rather than reconfigurations)